exercicios/para-casa/CrisPereira_exercicio36_S04_profBelisa.py

```diff
@@ -0,0 +1,30 @@
+# Semana 04 - exercício da Semana
+# Prof Belisa - estudante Cris Pereira
+
+# Ex. 36 
+# Desenvolva um programa que faça a tabuada de um número qualquer inteiro que será digitado pelo usuário, mas a tabuada não deve necessariamente iniciar em 1 e terminar em 10, o valor inicial e final devem ser informados também pelo usuário. Obs: Você deve verificar se o usuário não digitou o final menor que o inicial.
+
+print("Programa Tabuada Infinita")
+
+def usuario():
+    num_tabuada = int(input("Montar a tabuada de: "))
+    inicio = int(input("Começar por: "))
+    fim = int(input("Terminar em: "))
+
+    return num_tabuada, inicio, fim
+
+numero_tabuada, inicio_tabuada, fim_tabuada = usuario()
+
+while fim_tabuada <= inicio_tabuada:
```

Comment:
Não precisava ser menor ou igual néh? Parei para pensar nisso depois, podia ser só menor mesmo e mostrar apenas um resultado se for igual...